.readthedocs.yaml

```
@@ -0,0 +1,32 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
```

Zino itself will not currently run on Python 3.12 (some of its dependencies won't import).  Although the docs are currently empty, if we wish to auto-import some of the modules during doc generation, we should do so on a Python version that Zino supports (i.e. 3.11, for now).

```suggestion
    python: "3.11"
```